src/main/java/io/snyk/jenkins/SnykStepBuilder.java

```diff
@@ -225,34 +226,26 @@ public boolean perform(@Nonnull AbstractBuild<?, ?> build, @Nonnull Launcher lau
         }
       }
 
-      FilePath snykTestReport = workspace.child(SNYK_TEST_REPORT_JSON);
-      FilePath snykTestDebug = workspace.child(SNYK_TEST_REPORT_JSON + ".debug");
-
       ArgumentListBuilder argsForTestCommand = buildArgumentList(snykExecutable, "test", envVars);
 
-      try (
-        OutputStream snykTestOutput = snykTestReport.write();
-        OutputStream snykTestDebugOutput = snykTestDebug.write()
-      ) {
-        log.getLogger().println("Testing for known issues...");
-        log.getLogger().println("> " + argsForTestCommand);
-        testExitCode = launcher.launch()
-          .cmds(argsForTestCommand)
-          .envs(envVars)
-          .stdout(snykTestOutput)
-          .stderr(snykTestDebugOutput)
-          .quiet(true)
-          .pwd(workspace)
-          .join();
-      }
-
-      String snykTestReportAsString = snykTestReport.readToString();
+      ByteArrayOutputStream testStdout = new ByteArrayOutputStream();
+      logger.println("Testing for known issues...");
+      logger.println("> " + argsForTestCommand);
+      testExitCode = launcher.launch()
+        .cmds(argsForTestCommand)
+        .envs(envVars)
+        .stdout(testStdout)
+        .stderr(logger)
+        .quiet(true)
+        .pwd(workspace)
+        .join();
```

Comment:
We can try using `start` instead of `join` and pipe stdout directly to snyk-to-html's stdin.